### ISSUE TITLE
JEM 2.1.5+ with php 7(and Joomla 3.5): zebra_php gives error

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#1040]
Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; Zebra_Image has a deprecated constructor in /srv/www/vhosts/kath-jugend.ch/subdomains/intra/httpdocs/components/com_jem/classes/Zebra_Image.php on line 46

for the rest I don't see error notices with jem alone

(there will be a problem with the jcomments extension not with our plugin)

**Comment by jojo12** [comment]
is this zebra_image problem or our problem. Don't know

**Comment by jojo12** [comment]
we should update our Zebra_Image to Zebra_Image_2.2.4. There is no problem. It seems, we use an old one! (Info from Zebra_Image Author. 
I exchanged it, after the error notice was away

**Comment by Hoffi1** [comment]
This "deprecated method warning" is easy to fix (by simply replacing all these function names by the word "__constructor") and (because it talks about a future change) a less important thing in my eyes. Of course, we should fix it, no question.
But I'm more worry about the other changes I don't know till now if we get hidden problems hard to find by top level tests. So I've the task to check these on top of my list.

Updating Zebra Image is a good idea. It just requires a bit time to look what's different and if this may cause problems or requires adaptions on our code.

**Comment by Hoffi1** [comment]
The messages are fixed. Also I checked all files regarding things listed in PHP manual as backward incompatible changes. I found no real problems. Only the error handling maybe non-optimal because more and more errors are reported by throwing an exception instead of returning an error code. that means that everwhere in the code try/catch blocks must be inserted. Otherwise the exceptions fall through the code until they are catched e.g. by Joomla showing a general error page.

Also the Zebra Image class is not updated to 2.2.4 yet so I keep this issue open.
